Tests/L0/MSBuild/Get-SolutionFiles.ResolvesAsterisks.ps1

```diff
@@ -1,14 +0,0 @@
-[cmdletbinding()]
```

Comment:
I used the MSBuild tests as a starting point since much of the logic is similar. I noticed it would be more appropriate to collapse a couple of the MSBuild tests into a single tests. That's why this file and the next file are showing up as deletes here.